kbase-extension/scss/partials/_rcp.scss

```diff
@@ -0,0 +1,52 @@
+// run-control-panel
```

Comment:
this is the bar on the cell with the 'Run'/action button, the status line, and the Configure / Info / Job status / Result tabs